pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.poetry]
+name = "stata_kernel"
+version = "1.12.3"
+description = "A Jupyter kernel for Stata. Works with Windows, macOS, and Linux."
+authors = ["Kyle Barron <kylebarron2@gmail.com>", "Mauricio Caceres"]
+license = "GPLv3"
+classifiers = [
+    'Development Status :: 3 - Alpha',
+    'Environment :: Console',
+    'Environment :: Web Environment',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Education',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+    'Operating System :: MacOS',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX :: Linux',
+    'Operating System :: Unix',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+]
+keywords = ["stata"]
+repository = "https://github.com/kylebarron/stata_kernel"
+documentation = "https://kylebarron.dev/stata_kernel/"
+
+[tool.poetry.dependencies]
+python = "^3.6.1"
+beautifulsoup4 = "^4.6.3"
+pandas = "^1.0"
+pygments = "^2.2"
+pexpect = "^4.6.0"
+requests = "^2.19.1"
+jupyter-client = "^5.2.3"
+IPython = "^6.5.0"
+ipykernel = "^4.8.2"
+packaging = "^17.1"
+pillow = ">=5.2.0"
+pywin32 = { version = ">=223", platform = "Windows" }
```

Comment:
In a separate PR we could update these dependencies. The `^` allows semver compatibility of dependent versions. https://python-poetry.org/docs/dependency-specification/#caret-requirements

Reply:
So for example the `IPython` dependency above might be problematic because `pip install stata-kernel` would downgrade the user's ipython dependency to `>=6.5,<7`

Reply:
@kylebarron That's interesting. I agree a separate PR ought to happen after migrating to poetry and gh workflows. I also have to figure out where to test out the dependencies. I think this can be merged once the workflows file is updated.